include/osg/OcclusionQueryNode

```diff
@@ -158,6 +155,10 @@ public:
     osg::StateSet* getQueryStateSet();
     const osg::StateSet* getQueryStateSet() const;
 
+    // override construction of querygeometry based on child bounds
+    void setCustomQueryGeometry( osg::QueryGeometry* qg);
+    const osg::QueryGeometry* getCustomQueryGeometry() const { if (getComputeBoundingSphereCallback()) return getQueryGeometry(); return 0; }
+
```

Comment:
Just assume it's a custom query geometry if a computeboundcallback has been setted